investigations/cava2/Makefile

```diff
@@ -14,14 +14,26 @@
 # limitations under the License.
 #
 
-.PHONY: all clean
+.PHONY: all clean coq
```

Comment:
Currently, some makefiles have `_CoqProject` marked as `.PHONY`, while others don't. I think all of them should have it marked as `.PHONY`, because this will cause `_CoqProject` to be regenerated on each run, which is desirable because otherwise you have to delete (or `touch`) it each time you add, delete, or rename files.